packages/blade-core/src/styles/Button/button.module.css

```diff
@@ -0,0 +1,335 @@
+/* Button Styles */
+.btn {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  border: none;
+  outline: none;
+  text-decoration: none;
+  overflow: hidden;
+  border-radius: var(--border-radius-medium);
+  border-style: solid;
+  transition-property: background-color, border-color, box-shadow, transform;
+  transition-timing-function: var(--motion-easing-standard);
+  transition-duration: var(--motion-duration-xquick);
+  font-family: var(--font-family-text);
+  font-weight: var(--font-weight-medium);
+  text-align: center;
+  cursor: pointer;
+  user-select: none;
+}
+
+.btn[disabled] {
+  cursor: not-allowed;
+}
+
+/* Variants */
+.primary {
+  border-width: 0;
+}
+
+.secondary {
+  border-width: var(--border-width-thin);
+}
+
+.tertiary {
+  border-width: 0;
+}
+
+/* Sizes */
+.xsmall {
+  min-height: 28px;
+  padding-left: var(--spacing-3);
+  padding-right: var(--spacing-3);
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.small {
+  min-height: 32px;
+  padding-left: var(--spacing-4);
+  padding-right: var(--spacing-4);
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.medium {
+  min-height: 36px;
+  padding-left: var(--spacing-6);
+  padding-right: var(--spacing-6);
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.large {
+  min-height: 48px;
+  padding-left: var(--spacing-6);
+  padding-right: var(--spacing-6);
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+/* Icon-only buttons */
+.icon-only {
+  padding: 0;
+  width: auto;
+  height: auto;
+}
+
+.icon-only.xsmall {
+  width: 28px;
+  height: 28px;
+}
+
+.icon-only.small {
+  width: 32px;
+  height: 32px;
+}
+
+.icon-only.medium {
+  width: 36px;
+  height: 36px;
+}
+
+.icon-only.large {
+  width: 48px;
+  height: 48px;
+}
+
+/* Color variants - Background colors */
+.color-primary.primary {
```

Comment:
should we follow the `<key>-<value>` naming convention everywhere?

Then this class will become `.color-primary.variant-primary` right?